webview/src/experiments/components/table/styles.module.scss

```diff
@@ -559,11 +555,12 @@ $bullet-size: calc(var(--design-unit) * 4px);
 }
 
 .experimentCell .rowActions .starSwitch svg {
-  fill: $row-action-icon-color;
+  fill: $row-action-star;
+}
 
-  .rowSelected & {
-    fill: $row-action-selected-icon-color;
-  }
+.experimentCell .rowActions .starSwitch[aria-checked='true'] svg {
```

Comment:
Does it need that specificity to apply the rule? I'd simply change it to `.starSwitch[aria-checked='true'] svg {` otherwise so that if we ever get a variant, we can override the rules without adding to much specificity to it.

Reply:
I just saw `.experimentCell .rowActions .starSwitch svg {` and that's why this selector is that long. If it's possible to cut them both. It's easier to change the markup afterward.